### DISCUSSION TITLE
Need clarification on API warning

### DISCUSSION THREAD

**Discussion by mljrg** [community/community#24845] [Apps, API and Webhooks]
<p>Hello,</p>
<p>in the page <a href="https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/" rel="nofollow noopener">Authenticating with GitHub Apps</a> we can read</p>
<p><strong>&gt; Warning:</strong> The API may change without advance notice during the preview period. Preview features are not supported for production use. If you experience any issues, contact <a href="https://github.com/contact" rel="nofollow noopener">GitHub Support</a>.</p>
<p>which is a warning very common in the API documentation.</p>
<p>Does this warning mean that the corresponding API endpoints cannot be used in a production GitHub or OAuth app?</p>
<p>Or does it mean that they can be used in a production app, but because they are in preview mode, they may change, and in that case, we may contact GitHub support to fix related issues with their use or find recommended alternatives?</p>
<p>Thanks in advance</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/mljrg">@mljrg</a>!</p>
<p>I think that largely it’s a personal decision in a lot of ways. You absolutely can use Preview features in production apps with the understanding that things may change suddenly which might break things in ways that we can’t fix for you. You can always contact GitHub Support for advice and workaround options if things do change on you, but there’s no guarantee that we’ll be able to get things working the way that you had them.</p>
<p>Does that help to clarify?</p>

**Comment by mljrg** [comment]
<p>Hi <a class="mention" href="/that-pat">@that-pat</a></p>
<p>That clarifies a lot, thanks!</p>

**Comment by Zeeman76** [comment]
<p>Thank you very much for every thing</p>
